app/src/test/shared/services/LocalServiceTests.ts

```diff
@@ -145,77 +150,98 @@ describe('Local service cache tests', () => {
 
     const sandbox = new Sandbox();
 
+    class TestService extends AbstractLocalService {
```

Comment:
Factored out logic shared between the tests to make the individual tests easier to read.